### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by juliaogris**
This is a first step in improving and updating the docs.
You can look at a live version at https://github.com/juliaogris/sysl/tree/update-readme just below the file listing.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -61,4 +61,4 @@ If ``sysl.proto`` is updated the above command needs to be re-run to update the
 
 Status
 ------
-Sysl is currently targeted at early adopters. The current focus is to improve usability, especially error messages and warnings, and documentation.
+Sysl is currently targeted at early adopters. The current focus is to improve documentation and usability especially error messages and warnings.
```
Comment (anzdaddy) [inline README.rst]: s/usability/usability,/

File: README.rst [hunk 2]
```diff
@@ -7,42 +7,58 @@ Sysl
...
 
   > python setup.py bdist_wheel --universal
 
-If you are behind a corporate proxy setting you might want to consider setting up ``pip.conf``:
+Consider using `virtualenv <https://virtualenv.pypa.io/en/stable/>`_ and `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/>`_ to isolate your environment.
+
+Extending Sysl
+--------------
+In order to easily reuse and extend Sysl across systems, the Sysl compiler translates Sysl files
+into an intermediate representation expressed as protocol buffer messages. These protobuf messages can be consumed in your favorite porgramming language and transformed to your desired output. In this way you are creating your own Sysl exporter.
```
Comment (anzdaddy) [inline README.rst]: s/porgramming/programming/

File: README.rst [hunk 3]
```diff
@@ -7,42 +7,58 @@ Sysl
...
+
+Using the `protoc compiler <https://developers.google.com/protocol-buffers/>`_ you can translate the definition file of the intermediate representation ``src/proto/sysl.proto`` into your preferred programming language in a one of step or on every build. You can then easily consume Sysl models in your programming language of choice  in a typesafe way without having to write a ton of mapping
```
Comment (anzdaddy) [inline README.rst]: s/ {2}/ /

File: README.rst [hunk 4]
```diff
@@ -7,42 +7,58 @@ Sysl
...
+boilerplate. With that you can create your own customer tailored output diagrams, source code, views, integrations or other desired outputs.
```
Comment (anzdaddy) [inline README.rst]: s/customer tailored/tailored/

File: README.rst [hunk 5]
```diff
@@ -7,42 +7,58 @@ Sysl
...
+
+In this projects several Python based exporters exist under ``src/sysl/exporters`` and the relevant Python protobuf definitions ``sysl_pb2.py`` have been created from ``sysl.proto`` with ::
```
Comment (anzdaddy) [inline README.rst]: s/projects/project,/

File: README.rst (hunk 3 above)
Comment (juliaogris) [reply README.rst]: ohoh, not quite sure what this means.

File: README.rst (hunk 1 above)
Comment (juliaogris) [reply README.rst]: Done.
see https://github.com/anz-bank/sysl/pull/18

File: README.rst (hunk 2 above)
Comment (juliaogris) [reply README.rst]: Done.
see https://github.com/anz-bank/sysl/pull/18

File: README.rst (hunk 4 above)
Comment (juliaogris) [reply README.rst]: Done.
see https://github.com/anz-bank/sysl/pull/18

File: README.rst (hunk 5 above)
Comment (juliaogris) [reply README.rst]: Done.
see https://github.com/anz-bank/sysl/pull/18

**Pull request closed without merging**